curriculum/challenges/english/01-responsive-web-design/applied-visual-design/modify-fill-mode-of-an-animation.english.md

```diff
@@ -46,7 +46,7 @@ tests:
   button:hover {
     animation-name: background-color;
     animation-duration: 500ms;
-    /* add your code below this line */
+    /* Add your code below this line */
 
     /* add your code above this line */
```

Comment:
If I understood the gist correctly, this comment has already been consolidated.  Shouldn't it match the case of `/* Add your code below this line */`?

Reply:
Yeah, for some reason, my script did not recognize it as the same comment.